Cargo.toml

```diff
@@ -20,22 +20,21 @@ include = [
 ]
 
 [dependencies]
-bytemuck = "1.12"
-raw-window-handle = "0.4"
-thiserror = "1.0"
-ultraviolet = "0.9"
-wgpu = "0.13"
+bytemuck = "1.12.1"
+raw-window-handle = "0.5.0"
+thiserror = "1.0.37"
+ultraviolet = "0.9.0"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wgpu = { version = "0.14.0", features = ["webgl"] }
```

Comment:
This feature is already included in the web example: https://github.com/parasyte/pixels/pull/312/files#diff-9b657ee98202349de6df432957197f4efc6115c95e488ba75064bb24f076d6e3R24 Is it needed here, too?

See also: #276. We can do either-or, but I don't see any reason to do both.